pages/index.js

```diff
@@ -77,6 +85,7 @@ export default class Home extends React.Component{
 
   setCategory = selectedCategory => {
```

Comment:
Let's take some time to refactor this one already
```
setCategory = selectedCategory => 
  this.state.selectedCategory !== selectedCategory && 
    window.localStorage.setItem('ddf-category', selectedCategory) ||  this.setState({selectedCategory, currentPage: 0})
```
This will execute both commands due the nature of the boolean operator `||`.

Reply:
Done